src/software/thunderscope/chicker/chicker.py

```diff
@@ -0,0 +1,186 @@
+import pyqtgraph as pg
+import software.thunderscope.constants as constants
+import software.thunderscope.common_widgets as common_widgets
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import (
+    QApplication,
+    QCheckBox,
+    QGridLayout,
+    QGroupBox,
+    QButtonGroup,
+    QMenu,
+    QPushButton,
+    QRadioButton,
+    QVBoxLayout,
+    QHBoxLayout,
+    QWidget,
+    QSlider,
+)
+
+
+class ChickerWidget(QWidget):
+    def __init__(self, parent=None):
+        super(ChickerWidget, self).__init__(parent)
+
+        """
+        The Chicker Widget grid is laid out in the following way:
+                    ┌────────┐     ┌──────┐      ┌──────┐
+                    │ Charge │     │ Kick │      │ Chip │
+                    └────────┘     └──────┘      └──────┘
+
+                    ┌────────┐   ┌──────────┐ ┌─────────┐
+                    │No Auto │   │Auto Kick │ │Auto Chip│
+                    └────────┘   └──────────┘ └─────────┘
+
+                    ┌───────────────────────────────────┐
+                    │           Geneva Slider           │
+                    └───────────────────────────────────┘
+
+                    ┌───────────────────────────────────┐
+                    │           Power Slider            │
+                    └───────────────────────────────────┘
+        """
+        # grid --> groupBox --> button/slider
+        grid = QGridLayout()
+        self.radio_buttons = QButtonGroup()
+
+        self.charge_button_list = common_widgets.create_button("Charge", False)
+        self.charge_button = self.charge_button_list[1]
+        grid.addWidget(self.charge_button_list[0], 0, 0)
+
+        self.kick_button_list = common_widgets.create_button("Kick")
+        self.kick_button = self.kick_button_list[1]
+        grid.addWidget(self.kick_button_list[0], 0, 1)
+
+        self.chip_button_list = common_widgets.create_button("Chip")
+        self.chip_button = self.chip_button_list[1]
+        grid.addWidget(self.chip_button_list[0], 0, 2)
+
```

Comment:
pretty sure we can put all these buttons in the same widget and same with the radio buttons